apps/vscode/package.json

```
@@ -1036,6 +1036,16 @@
           "default": true,
           "markdownDescription": "When using a venv or conda environment, prefer Quarto CLI installed with pip in that environment. This will override Quarto CLI in the `PATH`, but not an explicitly configured `#quarto.path#`."
         },
+        "quarto.defaultEditor": {
+          "order": 11,
+          "type": "string",
+          "markdownDescription": "Default editor",
```

```suggestion
          "markdownDescription": "Default editor. Only applicable in Positron.",
```

This will be user facing in VS Code too so I think we need to say _something_ unless we can figure out how to make it work there too